cirq-core/cirq/sim/operation_target.py

```diff
@@ -68,7 +68,7 @@ def apply_operation(self, op: 'cirq.Operation'):
         protocols.act_on(op, self)
 
     @abc.abstractmethod
-    def copy(self: TSelfTarget) -> TSelfTarget:
+    def copy(self: TSelfTarget, deep_copy_buffers: bool = True) -> TSelfTarget:
```

Comment:
Update the docstring here too since it's the base interface.

Reply:
Updated.